### ISSUE TITLE
A11y_AzureDataStudio_AddNewController_ScreenReader : Voiceover is not announcing the label associated with "Authentication type" combo box .

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7804]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
GitHubTags:#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;#WCAG3.3.2;#DesktopApp;#VoiceOver;#SQL Azure Data Studio;#Mac

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS:macOS Mojave Version 10.14.6(18G87)

**Additional Details:**
Tool Name : VoiceOver

**Steps to Reproduce:**
1. Install and Launch the application the azure data studio application.
2. Connect to the servers with the required credentials.
3. Navigate to the "SQL Server Big Data Cluster" and expand it.
4. Navigate to the "SQL Server Big Data Cluster controller" and click on it.
5. Start VoiceOver.
6. Navigate till "Authentication type" combo box  using tab.
7. Observe the announcement by screen reader.

**Actual:**
Screen reader is  announcing "Azure Data studio , preview notebook-editor-0, windows, Basic Collapsed " for  "Authentication Type" combo box.

**Expected:**
Screen reader should announce label associated with  "Authentication Type" combo box.
For example for "Basic" list item , it should announce like "Authentication Type Basic "  

**User Impact:**
Screen reader user will find it difficult to understand the purpose of the control.

**Attachment for Reference:**
[7804_A11y_AzureDataStudio_AddNewController_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3742715/7804_A11y_AzureDataStudio_AddNewController_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
This has been fixed on insiders

**Comment by 12-shweta** [comment]
Verified the bug in below environment :
Version: 1.15.0-insider (user setup)
Commit: 041f34b
Date: 2020-01-08T05:36:19.161Z
VS Code: 1.38.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: MacOs version 10.15.2 (19C57)
Issue is fixed and does not reproduce .
![image](https://user-images.githubusercontent.com/54656883/72730340-39067580-3bb7-11ea-8c8d-fb7314748995.png)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_Mac;

**Comment by Rama8345** [comment]
A11ySev2

**Comment by Samhitha66** [comment]
#Closed; #Regressed:07-14-23;
